spec/jobs/irs_attempts_events_batch_job_spec.rb

```diff
@@ -41,23 +41,31 @@
       end
 
       it 'batches and writes attempt events to an encrypted file' do
-        result = IrsAttemptsEventsBatchJob.perform_now(timestamp: start_time, dir_path: @dir_path)
-        expect(result[:file_path]).not_to be_nil
+        expect_any_instance_of(described_class).to receive(:create_and_upload_to_attempts_s3_resource)        
```

Comment:
I think the plan last week was to have a `.and_return` on this stubbing out what would be returned. Does that sound right?